install-big.c

```diff
@@ -75,29 +75,29 @@ void hier()
-  c(auto_qmail,"doc","UPGRADE",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","SENDMAIL",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL.alias",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL.ctl",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL.ids",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL.maildir",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL.mbox",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","INSTALL.vsm",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","TEST.deliver",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","TEST.receive",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","REMOVE.sendmail",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","REMOVE.binmail",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.local2alias",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.local2ext",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.local2local",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.local2rem",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.local2virt",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.nullclient",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.relaybad",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.relaygood",auto_uido,auto_gidq,0644);
-  c(auto_qmail,"doc","PIC.rem2local",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","FAQ.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","UPGRADE.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","SENDMAIL.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.alias.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.ctl.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.ids.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.maildir.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.mbox.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","INSTALL.vsm.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","TEST.deliver.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","TEST.receive.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","REMOVE.sendmail.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","REMOVE.binmail.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.local2alias.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.local2ext.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.local2local.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.local2rem.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.local2virt.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.nullclient.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.relaybad.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.relaygood.md",auto_uido,auto_gidq,0644);
+  c(auto_qmail,"doc","PIC.rem2local.md",auto_uido,auto_gidq,0644);
```

Comment:
If we merged PR #22, either separately or as part of PR #15, we could avoid this part of the patch.

Reply:
#15 is 1.07, but #22 is currently 1.08 (odd). If we promote it to 1.07 (I'd agree), then yes.